example/programs/topic-subscriber/src/models.rs

```diff
@@ -0,0 +1,21 @@
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct MomentoModel {
+    pub key_one: String,
+    pub key_two: String,
+    pub key_three: i64,
+    #[serde()]
+    #[serde(rename(deserialize = "timestamp"))]
+    pub published_timestamp: DateTime<Utc>,
+}
+
+impl MomentoModel {
+    pub fn time_between_publish_and_received(&self) -> i64 {
```

Comment:
If this is named time_between, you should return a std::time::Duration. If this is milliseconds between publish and received, it should be named milliseconds_between_publish_and_received. “Time” does not mean “milliseconds” to many minds, and on ec2 some calls may report 0 given that you’re using i64 milliseconds. Without the unit, it may be confusing to see 0’s printed.

Reply:
I'll rename.  But there is not a chance of 0 here.   However, I agree on the naming.  Made it "milliseconds" over time